manifests/4.3/cluster-logging.v4.3.0.clusterserviceversion.yaml

```diff
@@ -20,46 +19,78 @@ metadata:
     olm.skipRange: ">=4.2.0 <4.3.0"
     alm-examples: |-
         [
-            {
-              "apiVersion": "logging.openshift.io/v1",
-              "kind": "ClusterLogging",
-              "metadata": {
-                "name": "instance",
-                "namespace": "openshift-logging"
-               },
-              "spec": {
-                "managementState": "Managed",
-                "logStore": {
-                  "type": "elasticsearch",
-                  "elasticsearch": {
-                    "nodeCount": 3,
-                    "redundancyPolicy": "SingleRedundancy",
-                    "storage": {
-                      "storageClassName": "gp2",
-                      "size": "200G"
-                     }
-                   }
-                },
-                "visualization": {
-                  "type": "kibana",
-                  "kibana": {
-                    "replicas": 1
+          {
+            "apiVersion": "logging.openshift.io/v1",
+            "kind": "ClusterLogging",
+            "metadata": {
+              "name": "instance",
+              "namespace": "openshift-logging"
```

Comment:
don't we want to have the annotation if we're going to have the `logforwarding` definition?

Reply:
> don't we want to have the annotation if we're going to have the `logforwarding` definition?

yes.  our guidelines require explicit opt into the feature.  @bparees should the example enable LF via annoation by default